mix.exs

```diff
@@ -27,8 +27,8 @@ defmodule BambooGmail.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:bamboo, "~> 1.3"},
-      {:goth, "~> 1.1.0"},
+      {:bamboo, "~> 2.1"},
```

Comment:
not quite sure how you want to tie the versions together. I'm not sure if the error handling will break for users of bamboo versions ~> 1.3

Reply:
I'll tag specific versions. This is fine for now.